godelegator/templates/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.deployment.name }}
+spec:
+  # type: ClusterIP
+  ports:
+    - port: 5050
```

Comment:
It would be nice to define the port 5050 as variable. It is used couple times in this PR.